test/integration/puppetlabs/services/jruby/request_handler_test.clj

```diff
@@ -63,25 +63,18 @@
                                                        (subs expected-md5 0 8))
                                           expected-md5
                                           "contents"])
-               ;; The 'text/plain' content-type mimics what a Puppet agent
-               ;; sends to a master for a file-bucket PUT.  Ideally, this
-               ;; should be "application/octet-stream" but including this at
-               ;; present would result in a Puppet Ruby error - Puppet Client
-               ;; sent a mime-type (application/octet-stream) that doesn't
-               ;; correspond to a format we support.
```

Comment:
If we want to doc anything, the currently remaining exceptional behavior is the `accept: binary` thing. :grapes: